tests/Testing/TestResponseTest.php

```diff
@@ -2807,7 +2808,10 @@ public function testAssertSessionMissing()
         $response->assertSessionMissing('foo');
     }
 
-    public function testAssertSessionMissingValue()
+    #[aTestWith(['foo', 'badvalue'])]
+    #[TestWith(['foo', null])]
+    #[aTestWith([['foo', 'bar'], null])]
```

Comment:
I assume, this is a typo:
```suggestion
    #[TestWith(['foo', 'badvalue'])]
    #[TestWith(['foo', null])]
    #[TestWith([['foo', 'bar'], null])]
```

Reply:
Thanks, forgot to update after debugging

Reply:
You're welcome